uml/backend-software-system/screen-handler.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+package "Screen Handler" {
+  class EventType {
+    type : String
+    sprite : Sprite
+  }
+
+  class SpritesReader {
+    +readSprites() : List<Sprite>
+  }
```

Comment:
This logic would be handled by the socket server component of the screen container.